src/components/common/NavBar.jsx

```diff
@@ -5,62 +5,94 @@ import { Link } from "react-router-dom";
 import LangDropdown from "./LangDropdown";
 
 export const NavBar = () => {
+  const [isHamburgerShown, setIsHamburgerShown] = React.useState("-100%");
+
+  const handlehamburger = () => {
```

Comment:
this branch has files from the other PR, please clean it up so i can review the actual changes